README.md

```diff
@@ -48,6 +50,33 @@ async def main():
     # have to wait delay period before message is going to be sent.
 ```
 
+### **Retries with `rabbitmq-delayed-message-exchange` plugin**
+
+To send delayed message you can install `rabbitmq-delayed-message-exchange`
```

Comment:
I guess for some people it might be confusing, and they might think that we cannot use delay feature without this plugin. 

Maybe it would be a little better to explain how delays work without that plugin and what are benefits in using plugin instead of our initial delay solution?

Reply:
The delay plugin can handle tasks with different delay times well, and the delay based on dead letter queue is suitable for tasks with the same delay time.
@chandr-andr  Hello, do you have time to improve these? 🤩

Reply:
@vvanglro Hi! Yeap, I'll apply all requested changes tomorrow. Sorry for the long feedback)